### PR TITLE
Update to grafana 5.3.4

### PR DESCRIPTION
**Pull request by eheydrick**
Signed-off-by: Eric Heydrick <eheydrick@gmail.com>

### Description

Update to latest grafana release, 5.3.4. Also add testing on Ubuntu 18.04 and drop Ubuntu 14.04. All tests pass.

### Issues Resolved

None.

### Check List
- [x] All tests pass. See https://github.com/sous-chefs/grafana/blob/master/TESTING.md
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          2 Errors
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please add labels to this Pull Request</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a <a href="https://github.com/sous-chefs/grafana/blob/master/CHANGELOG.md">CHANGELOG</a> entry.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: .kitchen.dokken.yml [hunk 1]
```diff
@@ -62,6 +62,13 @@ platforms:
     intermediate_instructions:
       - RUN /usr/bin/apt-get update
 
+- name: ubuntu-18.04
+  driver:
+    image: dokken/ubuntu-18.04
+    pid_one_command: /bin/systemd
+    intermediate_instructions:
+      - RUN /usr/bin/apt-get update
+
```
Comment (damacus) [inline .kitchen.dokken.yml]: Whilst you're here you can probably remove Ubuntu 14.04

**Review by damacus — CHANGES_REQUESTED** [review]
Remove Ubuntu 14.04 from dokken?

**Comment by damacus** [comment]
Hah Danger actually picked up something useful, you also probably need a CHANGELOG entry for the version upgrade.

**Comment by eheydrick** [comment]
I'll nuke 14.04 and add a CHANGELOG entry, Where should it go? An Unreleased section at the top?

**Comment by xorima** [comment]
@eheydrick did you get a chance to add the Change log and remove 14.04?

**Comment by eheydrick** [comment]
Oh apologies, I'll add that tonight.

**Comment by eheydrick** [comment]
Ready for final review. I also bumped to the latest release, 5.3.4. There's a security issue fixed in 5.3.3 so good to be current.

**Review by xorima — APPROVED** [review]
Looks good to me, thanks

**Review by damacus — APPROVED** [review]

**Pull request merged**